s3downloader.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+import boto3
+import json
+
+log = logging.getLogger('s3_downloader')
+log.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+# create formatter and add it to the handlers
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+# add the handlers to the logger
+log.addHandler(ch)
+
+endpoint_url = os.environ.get('S3_ENDPOINT_URL', 'http://localhost:9000')
```

Comment:
add readme to explain how it works and what keys need to be supplied

Reply:
fixed in b45b9bb